src/style.css

```
@@ -1 +1,66 @@
-/* Styles go here */
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.header {
+  padding-top: 0;
+  padding-bottom: 0;
+  padding-left: 50px;
+  padding-right: 50px;
+  height: 60px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.image__logo {
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  color: #000;
+  position: relative;
+  text-decoration: none;
+  text-transform: uppercase;
+  line-height: 60px;
+  padding: 23px 0;
+  margin: 0 10px;
+  transition: color ease 0.5s;
+}
+
+#last_link {
+  margin-right: 0;
+}
+
+#first_link {
+  margin-left: 0;
+}
+
+.nav__link:hover,
+.is-active {
+  color: #00acdc;
+}
+
+.nav__link:hover:not([data-qa="hover"]):after,
+.is-active:after {
```

You don't need to use pseudo-element `after` to all links.
```suggestion
.is-active:after {
```

---

thanks